priv/auth-ui/en.json

```diff
@@ -21,23 +21,29 @@
     "ask_new_invite": "Ask the person who invited you to the organization to send you another invite.",
     "email_label": "Email address",
     "expired_invitation": "This invitation has expired.",
-    "invited_you": "%{inviter} has invited you to the %{orgName} organization.",
+    "expired_invitation_get_in_touch": "This invitation has expired. Get in touch with %{inviterName} if you have questions.",
     "full_name_label": "Full name",
     "full_name_placeholder": "Let's put a name to that face",
+    "invited_you": "%{inviter} has invited you to the %{orgName} organization.",
     "join_now": "Join now by signing in with your Qzzr account.",
+    "login_error": "Incorrect info. Try entering it again.",
     "login_name": "LOG IN",
+    "login_to_existing_hub": "Log in to your existing account to join the %{hubName} Hub",
+    "login_to_existing_org": "Log in to your existing account to join the %{hubName} Organization",
     "password_confirm_label": "Confirm new password",
     "password_confirm_placeholder": "New password",
     "password_label": "Password",
     "password_placeholder": "8 characters minimum",
-    "signup_name": "SIGN UP",
     "signup_free": "First sign up for a free Boombox account.",
-    "submit_signin": "SIGN IN",
-    "submit_signup": "SIGN UP & CONTINUE",
+    "signup_name": "SIGN UP",
+    "signup_to_join_hub": "Signup to join the %{hubName} Hub",
+    "signup_to_join_org": "Signup to join the %{hubName} Organization",
+    "submit_signin": "LOG IN",
```

Comment:
Also, text transformation should happen in CSS. All these that are all capitals should just have the first letter capital